UofS-Pulse-Binfo/analyzedphenotypes
# Phenotype summary fields formatter

This PR will create a field group formatter for displaying a set of field types showing quantitative values.
![Screen Shot 2019-05-15 at 2 25 15 PM (2)](https://user-images.githubusercontent.com/15472253/57806895-56786b00-771d-11e9-874c-72ae569cd119.png)

## Metadata

 - Issue # https://github.com/UofS-Pulse-Binfo/analyzedphenotypes/issues/81
Phenotype Summary Fields



- [X] This PR is dependent upon [PR #86 ]
https://github.com/UofS-Pulse-Binfo/analyzedphenotypes/pull/86

Documentation:

## Description



## Dependencies



## Testing?




1. For testing, this formatter can be used to format phenotype summary field types in the required PR.
2. Start by adding field to Trait content type. Ideally, at least 2 field types and max of 5 field type but it can support 6 and up. Please note that the name of the field is the name/label shown in the frontend.
3. In Manage Display, create a group and set the Format to Analyzed Phenotype Data Summary.
Save and move this field out of the disabled list.
4. With the new field, start adding the field types added in step #2 as shown in the example below.
![Screen Shot 2019-05-15 at 2 35 44 PM (2)](https://user-images.githubusercontent.com/15472253/57807631-caffd980-771e-11e9-9581-3952f6b1920f.png)
5. Save, clear cache and reload page.

OPTIONS:
1. The entire field group can be added to field set to Tripal Pane format in which case it will show a pane with a corresponding navigation and operates like any Tripal pane content.
2. On its own - without navigation.
3. Tested on Frameset, Div and HTML Element.

ORDER OF PLACEMENT:
From left to right, from one level to next. The top most level or the first level can have 3 items only when the item count is odd otherwise it will distribute items equally to each level. Animation may also show this ordering rule.

![Screen Shot 2019-05-15 at 2 25 15 PM (2)](https://user-images.githubusercontent.com/15472253/57808850-86c20880-7721-11e9-82f6-51f95046e2d3.png)
